src/api/weth/WethApiMock.ts

```diff
@@ -0,0 +1,76 @@
+import { logDebug } from 'utils'
+import { Receipt } from 'types'
+import { waitAndSendReceipt } from 'utils/mock'
+import { WethApi, WrapUnwrapParams } from './WethApi'
+
+import { RECEIPT } from '../../../test/data'
+
+// interface WethApiMockParams {
```

Comment:
~Bunch of dead code~

I know I know, will wait for next PR

Reply:
Yes, I should have cleanup before sending. But I wanted to have it for your review and it was pretty late already! thanks for understanding :)